doc/release/1.15.0-notes.rst

```diff
@@ -12,6 +12,8 @@ New functions
 
 * `np.gcd` and `np.lcm`, to compute the greatest common divisor and least
   common multiple.
+* `np.ma.stack`, the `np.stack` array joining function generalized to masked
```

Comment:
super nit: "array joining" should be hyphenated as "array-joining"

Reply:
Haha, sure, fixed!